Cargo.toml

```diff
@@ -22,6 +22,11 @@ rand = ["bitcoin/rand"]
 bitcoin = "0.28.0"
 serde = { version = "1.0", optional = true}
 
+[dev-dependencies]
+bitcoind = {version = "0.26.1", features=["22_0"]}
+actual-rand = { package = "rand", version = "0.8.4"}
+bitcoin = { version = "0.28", features = ["rand"]}
```

Comment:
since there is rand anyway, can't we use `sign_schnorr_with_aux_rand` and remove the feature rand so that we don't have both bitcoin and bitcoin-with-rand?

Reply:
Adressing in a follow-up PR.